README.md

```diff
@@ -1,39 +1,120 @@
-# Ziti SDK for Python
+<a href="https://docs.openziti.io/">
+    <img src="./images/ziti-logo-dark.svg" alt="ziti logo" title="OpenZiti" align="right" height="60" />
+</a>
 
-## Installation
+# Python SDK for OpenZiti
```

Comment:
Definitely would prefer to see this as markdown. sometimes for images you really need to embed html on the markdown so that you can style an image but most of what I see here should be markdown, not html

Reply:
To get the styling that I wanted (a lot of it simply centering and/or centering inline content), I had to use HTML to get it to work how I wanted. Much of the HTML structure is copied from other READMEs that I drew inspiration from, and I did remove HTML from the copied content wherever possible.

I can revisit the HTML and try to make more of it work as markdown, but it was similar to this image. After a decent amount of effort, I couldn't get the markdown to do what I wanted, and it easily worked as HTML, so I didn't want to waste more time trying to get it to work when it may not have even been possible and worked as is.
![image](https://user-images.githubusercontent.com/95235264/232067402-25694fec-e50e-4ec2-9d59-0ee572ffc057.png)